src/utils.js

```diff
@@ -12,6 +12,43 @@ import { PACKAGE_TYPES, LOCAL_PROTOCOLS } from 'const';
 /* global nodeRequire, localesRoot */
 
 const SOURCE_MAP_RE = /\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/;
+// Firefox's version format is laxer than Chrome's, it accepts:
```

Comment:
Nit, we may mention this is valid only for MV2 addons (e.g. `Firefox's version format for MV2 extensions is laxer than chrome's...`)

Reply:
done